ingestion/src/main/java/feast/ingestion/ImportJob.java

```diff
@@ -152,7 +152,8 @@ public static PipelineResult runPipeline(ImportOptions options) throws IOExcepti
     PCollectionList<FeatureSetReference> sinkReadiness = PCollectionList.empty(pipeline);
 
     for (Store store : stores) {
-      FeatureSink featureSink = getFeatureSink(store);
+      FeatureSink featureSink =
+          getFeatureSink(store, options.getEnableRedisTTL(), options.getMaxRedisTTLJitterSeconds());
```

Comment:
we have sink options grouped per specific type (see `protos/feast/core/Store.proto`) and this one is generic API. Those options should be passed via `RedisConfig` & `RedisClusterConfig`

Reply:
@ravdin To restate what Oleksii said, you should be putting this configuration here: https://github.com/feast-dev/feast/blob/master/protos/feast/core/Store.proto#L108

Reply:
Done.